sample-lnd.conf

```diff
@@ -1489,6 +1489,11 @@ litecoin.node=ltcd
 ; window to allow more inputs to be added and thereby lower the fee per input.
 ; sweeper.batchwindowduration=30s
 
+; The max fee rate can be used when sweeping funds. Setting this value too low
```

Comment:
Should be mentioned that the unit is sat/kw. Otherwise it can be confusing

Reply:
switched to sat/vb